packages/appium/lib/appium.js

```diff
@@ -443,9 +444,10 @@ class AppiumDriver extends BaseDriver {
     // The tricky part is that because we support command plugins, we need to wrap any of these
     // cases with plugin handling.
 
-    const isGetStatus = cmd === 'getStatus';
+    const isGetStatus = cmd === GET_STATUS_COMMAND;
+    const isDeleteSession = cmd === DELETE_SESSION_COMMAND;
     const isUmbrellaCmd = !isGetStatus && isAppiumDriverCommand(cmd);
```

Comment:
what's an "umbrella command"?

Reply:
this means the command must be executed by the specific downstream platform driver rather than the AppiumDriver one

Reply:
it's the other way around, @mykola-mokhnach . the "umbrella" driver is the one defined here, AppiumDriver in appium.js. the "session" driver is the one that will handle most commands. There's a distinction because some commands (createSession, deleteSession) are handled by Appium initially. They don't get executed by the session driver immediately because Appium itself manages its own list of session drivers and this needs maintenance. Once a session exists, commands (like findElement or whatever) are executed directly by the session driver; this AppiumDriver class doesn't define logic for them at all.